docs/evaluate/temporal-cloud/service-availability.mdx

```diff
@@ -51,6 +53,18 @@ Currently, Temporal Cloud operates in several regions on Amazon Web Services (AW
 
 _**Your Workers and Client code aren't required to be hosted on AWS.**_
 
+### GCP
+
+Temporal Cloud operates in two regions on Google Cloud (GCP):
+
+| Area          | Code                 | Region            |
+| ------------- | -------------------- | ----------------- |
+| North America | us-west1             | Oregon            |
+| Asia Pacific  | australia-southeast1 | Sydney            |
+
+
+_**Your Workers and Client code aren't required to be hosted on GCP.**_
```

Comment:
Can this statement be put in a more generic way in the page?
e.g. from the sentence just above the split between AWS and GCP, "here are the regions that Temporal Cloud can provide a server in, but your Workers and Client code is not required to be hosted in the same region, of the same cloud provider. We however advise on creating your Namespace in a region geographically close to your Workers' hosting location."

Reply:
Should we just remove this? We also have it for AWS. The above paragraph covers the point, so we don't necessarily need this in this location.

Reply:
Works for me